train.py

```diff
@@ -84,14 +84,14 @@
     ),
     LanguageCleaning(
         detected_languages=args.detected_languages(),
-        preferred_languages=args.preferred_languages(),
+        preferred_languages=args.preferred_languages(), #IS this set to Eng and None?
```

Comment:
It'll detect English, French, German and Spanish.

If the detected language is English it'll keep the English description.

If it's None it'll keep the unknown language description

For the other detected languages it'll filter those out